packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -29,6 +32,12 @@ export interface WizardToggleProps {
   mainAriaLabel?: string;
   /** If the wizard is in-page */
   isInPage?: boolean;
+  /** Flag indicating the wizard has a drawer for at least one of the wizard steps */
```

Comment:
I would mark all props for new feature beta.

Reply:
Ah yes, good idea! Updated